README.md

```diff
@@ -1,2 +1,65 @@
-# homebrew-temple
-Homebrew Formulae for Temple
+# TempleEight Homebrew Tap
+
+This is a [Homebrew][brew] tap for formulae for software developed by TempleEight.
+
+
+## Setup
+
+Using these formulae requires Homebrew, which in turn requires Xcode Command Line Tools. If you
+have not yet installed Homebrew, a quick summary is at the end of this
+document.
+
+Once homebrew is installed, simply run:
+
+    brew tap templeeight/temple
+
+
+## Use
+
+To install software, just use `brew install` with the name of the formula. You
+may wish to run `brew update` before hand to get the latest version of the
+formulae. For example, to install the latest version of Temple:
+
+    brew update
+    brew install temple
+
+To update brew taps:
+
+    brew update
+
+To upgrade all software installed with Homebrew:
+
+    brew upgrade 
+
+To update just temple:
+
+    brew upgrade temple  
+
+
+## Appendix: overview of installing Homebrew
+
+The Homebrew developers suggest installing Homebrew at `/usr/local` to maximize
+compatibility with existing software. To do so, follow the instructions on
+[their website][brew].
+
+This author prefers `/opt/homebrew`, finding that it works well enough in
```

Comment:
Why do we need our own method of installing brew? The one-liner on the website should be good enough for beginners, and non-beginners will already have it installed